Makefile

```diff
@@ -130,7 +130,7 @@ notice: python-env
 .PHONY: python-env
 python-env:
 	@test -d $(PYTHON_ENV) || virtualenv $(VIRTUALENV_PARAMS) $(PYTHON_ENV)
-	@$(PYTHON_ENV)/bin/pip install -q --upgrade pip autopep8 six
+	@$(PYTHON_ENV)/bin/pip install -q --upgrade pip autopep8==1.3.5 six
```

Comment:
I think we should not upgrade `autopep8` and `six` here.
We need to provide a `requirements.txt` and let `pip` install every dependency based on that file.